eng/Versions.props

```diff
@@ -188,8 +188,8 @@
     <MicrosoftBuildUtilitiesCoreVersion>16.9.0</MicrosoftBuildUtilitiesCoreVersion>
     <MicrosoftBuildLocatorVersion>1.2.6</MicrosoftBuildLocatorVersion>
     <MicrosoftBuildUtilitiesCoreVersion>16.9.0</MicrosoftBuildUtilitiesCoreVersion>
-    <MicrosoftCodeAnalysisCommonVersion>4.0.0-2.21354.7</MicrosoftCodeAnalysisCommonVersion>
-    <MicrosoftCodeAnalysisCSharpVersion>4.0.0-2.21354.7</MicrosoftCodeAnalysisCSharpVersion>
+    <MicrosoftCodeAnalysisCommonVersion>4.0.0-2.21322.50</MicrosoftCodeAnalysisCommonVersion>
+    <MicrosoftCodeAnalysisCSharpVersion>4.0.0-2.21322.50</MicrosoftCodeAnalysisCSharpVersion>
```

Comment:
Why are you downgrading these versions❔ We need at least something that supports `LanguageVersion.CSharp10`.

``` text
src/Razor/Microsoft.CodeAnalysis.Razor/src/RazorProjectEngineBuilderExtensions.cs(93,62): error CS0117: (NETCORE_ENGINEERING_TELEMETRY=Build) 'LanguageVersion' does not contain a definition for 'CSharp10'
```

Reply:
Put another way, OmniSharp is not our only driver here

Reply:
Yeah, I think I'll need to bump up the dependencies in O# instead. Not sure what gremlins that will reveal but oh-well.